### PR TITLE
[Cache] Reduce the automatic TrieNodeCache FastCache size 

### PR DESCRIPTION
**Pull request by hyunsooda**
## Proposed changes

Fastcache size has been reduced to a maximum of 10 gigabytes.

## Types of changes

Please put an x in the boxes related to your change.

- [ ] Bugfix
- [ ] New feature or enhancement
- [ ] Others

## Checklist

*Put an x in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.*

- [ ] I have read the [CONTRIBUTING GUIDELINES](https://github.com/klaytn/klaytn/blob/master/CONTRIBUTING.md) doc
- [ ] I have signed the [CLA](https://cla-assistant.io/klaytn/klaytn)
- [ ] Lint and unit tests pass locally with my changes (`$ make test`)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Related issues

- Please leave the issue numbers or links related to this PR here.

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Review by blukat29 — DISMISSED** [review]

File: storage/statedb/database.go [hunk 1]
```diff
@@ -336,14 +336,9 @@ func getTrieNodeCacheSizeMiB() int {
 		return 0
 	} else if totalPhysicalMemMiB < 20*1024 {
 		return 1 * 1024 // allocate 1G for small memory
+	} else {
+		return 10 * 1024 // allocate 10G for large memory (>20G)
 	}
```
Comment (aidan-kwon) [inline storage/statedb/database.go:341]: I think 10GB is too big for 20GB ~ 30GB memory. 
How about the following?
```suggestion
	} else if totalPhysicalMemMiB < 30*1024 {
		return 6 * 1024 // allocate 6G for medium memory
	} else {
		return 10 * 1024 // allocate 10G for large memory (>20G)
	} 
```

Comment (hyunsooda) [reply storage/statedb/database.go:341]: Ah, I've made a significant mistake. Thank you for pointing that out. The only minor adjustment I'd like to make is to reduce the number from 6 to 3. Does that sound acceptable to you? (becuause it seems to jump sharply)

Comment (aidan-kwon) [reply storage/statedb/database.go]: I tried to maintain the original value is possible. Originally, 6GB is assigned for 20GB of physical memory (20GB * 0.3).

**Review by aidan-kwon — APPROVED** [review]

File: storage/statedb/database.go (hunk 1 above)
Comment (hyunsooda) [reply storage/statedb/database.go]: I see. thanks.

**Review by blukat29 — APPROVED** [review]

**Pull request merged**